pkg/virtctl/vm/migrate.go

```diff
@@ -33,6 +34,8 @@ import (
 
 const COMMAND_MIGRATE = "migrate"
 
+var nodeSlectorLabels []string
```

Comment:
Ideally, I'd like to see this var in a struct for this command and not as global variable.

Reply:
done